src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the start of the class definition.